src/plugins/intel_gpu/include/intel_gpu/primitives/non_max_suppression.hpp

```diff
@@ -17,6 +17,12 @@ namespace cldnn {
 struct non_max_suppression : public primitive_base<non_max_suppression> {
     CLDNN_DECLARE_PRIMITIVE(non_max_suppression)
 
+    enum Rotation {
+        NONE,
+        CLOCKWISE,
+        COUNTERCLOCKWISE
+    };
```

Comment:
How about share this with ov_ops?
It is using integer in nms_ie_internal.hpp but seems that we can declare one enum and use it from plugin.

Reply:
I thought of this, but decided not to introduce new enum in ```attr_types.hpp```. This enum is not a part of interface (i.e., it is not used in NMSRotated op - there are boolean ```clockwise``` attribute instead), it is just a detail of implementation of internal operation ```NonMaxSuppressionIEInternal```.